### PR TITLE
[develop]: Update PR & issue templates, Contributor's Guide

### PR DESCRIPTION
**Pull request by gspetro-NOAA**
## DESCRIPTION OF CHANGES: 
This PR updates the SRW App's PR & Issue templates to align more with best practices reflected in other UFS/NOAA repositories (e.g., WM, global-workflow). This includes addition of checklists and links to the Contributor's Guide and Code Manager's Guide in the wiki. 

This PR also updates the Contributor's Guide chapter of docs to reflect updates added to the wiki, including information on testing with Jenkins. 

Other minor updates include adding "areas of expertise" for code managers and widening the RTD view to make table/code views easier. 

## TESTS CONDUCTED: 
N/A

## DEPENDENCIES:
N/A

## DOCUMENTATION:
N/A

## ISSUE (optional): 
Issue #307 
Issue #334

### CODE REVIEW

File: docs/UsersGuide/source/ContributorsGuide.rst [hunk 1]
```diff
@@ -27,67 +27,66 @@ Scientists from across multiple labs and organizations have volunteered to revie
...
+    +==================+================================================+===================================================================================+
+    | EMC              | Chan-Hoo Jeon (@chan-hoo)                      |                                                                                   |
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Ben Blake (@BenjaminBlake-NOAA)                | Output visualization                                                              |
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Ratko Vasic (@RatkoVasic-NOAA)                 |                                                                                   |
+    +------------------+------------------------------------------------+-----------------------------------------------------------------------------------+
```
Comment (JeffBeck-NOAA) [inline docs/UsersGuide/source/ContributorsGuide.rst:38]: You can add EMC for Ben and Ratko here.

File: docs/UsersGuide/source/ContributorsGuide.rst [hunk 2]
```diff
@@ -27,67 +27,66 @@ Scientists from across multiple labs and organizations have volunteered to revie
...
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Jong Kim (@jkbk2004)                           | UFS Weather Model configuration, forecast sensitivity analysis, data assimilation |
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Natalie Perlin (@natalie-perlin)               | Generic Linux/Mac installations, hpc-stack/spack-stack                            |
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Gillian Petro (@gspetro-NOAA)                  | Documentation                                                                     |
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
```
Comment (JeffBeck-NOAA) [inline docs/UsersGuide/source/ContributorsGuide.rst:46]: Need to add affiliations for Jong Kim, Natalie, and someone named Gillian.

File: docs/UsersGuide/source/ContributorsGuide.rst [hunk 3]
```diff
@@ -27,67 +27,66 @@ Scientists from across multiple labs and organizations have volunteered to revie
...
+    |                  | Daniel Abdi (@danielabdi-noaa)                 | Workflow generation, testing RRFS on the cloud, environment modules               |
+    +------------------+------------------------------------------------+-----------------------------------------------------------------------------------+
+    | NCAR             | Mike Kavulich (@mkavulich)                     | CCPP/physics                                                                      |
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Will Mayfield (@willmayfield)                  | Verification/METplus tasks, regional_workflow (esp. on Cheyenne)                  |
+    +------------------+------------------------------------------------+-----------------------------------------------------------------------------------+
```
Comment (JeffBeck-NOAA) [inline docs/UsersGuide/source/ContributorsGuide.rst:66]: NCAR for Will

File: docs/UsersGuide/source/ContributorsGuide.rst [hunk 4]
```diff
@@ -27,67 +27,66 @@ Scientists from across multiple labs and organizations have volunteered to revie
...
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Christina Holt (@christinaholtNOAA)            | Workflow, conda environment support, and testing                                  |
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Christopher Harrop (@christopherwharrop-noaa)  | Rocoto                                                                            |
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Daniel Abdi (@danielabdi-noaa)                 | Workflow generation, testing RRFS on the cloud, environment modules               |
+    +------------------+------------------------------------------------+-----------------------------------------------------------------------------------+
```
Comment (JeffBeck-NOAA) [inline docs/UsersGuide/source/ContributorsGuide.rst:62]: You can add GSL for the above five people.

**Review by JeffBeck-NOAA — APPROVED** [review]
Excellent additions!  Thanks!

**Review by panll — APPROVED** [review]
Looks good to me, thanks!

File: docs/UsersGuide/source/ContributorsGuide.rst (hunk 4 above)
Comment (gspetro-NOAA) [reply docs/UsersGuide/source/ContributorsGuide.rst:62]: On [RTD](https://srw-ug.readthedocs.io/en/feature-templates/ContributorsGuide.html), GSL actually applies to all 5 of them (and NCAR to Will), tho I can see how that might be confusing looking through the .rst files without looking at RTD. Do you want me to change it to list organization on every line?

Comment (JeffBeck-NOAA) [reply docs/UsersGuide/source/ContributorsGuide.rst:62]: Oh!  Sorry, I didn't realize things look just fine in RTD.  No need to make any modifications, as it looks good!

**Pull request merged**